plugin/pkg/admission/webhook/gencerts.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+# gencerts.sh generates the certificates for the generic webhook admission plugin tests.
```

Comment:
This is an interesting case.  This was a utility script used by the `ImagePolicyWebhook` and in various other places, including `pkg/util/webhook`.  I think this script should be put into `hack` or something so that people can call it as a helper when they need to generate these for tests.  Thoughts?